word_racer/session.py

```diff
@@ -0,0 +1,295 @@
+            self.round_scores = Counter()
+            self._gen_board()
+            self._get_score_dict()
+            self._gen_image()
+            await asyncio.sleep(3)
+
+            # send board image
+            f = discord.File(str(self.output_image_path))
+            await self.ctx.send(f"Starting round {self.level+1}. {len(self.valid_words)} words to find.",file=f)
+
+            # Message handler for round
+            self.round_finish = False
+            await self.run_round()
+
+            # Round cleanup
+            await self.finish_round()
+            self.level += 1
+            
+        await self.end_game()
+
+    async def _send_startup_msg(self):
+        await self.ctx.send("Starting Word Racer. Find words boggle-style and gain points."
+                            " Incorrect calls are -1 point. In rounds 2-4 there are bonuses:"
+                            " blue is 2x points and red is 3x points (they multiplicatively stack)."
+                            " Good luck!")
+
+    async def run_round(self):
+        timer_task = self.ctx.bot.loop.create_task(self.timer_task())
+        reaction_task = self.ctx.bot.loop.create_task(self.reactions_handler())
+        try:
+            await self.ctx.bot.wait_for("message", check=self.check_message, timeout=120)
+        except asyncio.TimeoutError:
+            #Round over
+            pass
+        finally:
+            timer_task.cancel()
+            reaction_task.cancel()
+
+
+    async def reactions_handler(self):
+        while not self.round_finish or self.reaction_queue:
+            if self.reaction_queue:
+                m = self.reaction_queue.pop(0)
+                await m[0].add_reaction(m[1])
+            else:
+                await asyncio.sleep(.25)
+
+    async def finish_round(self):
+        await self.send_round_table(f"- Round {self.level+1} over! \n Round scores:")
+        if self.level in [1,2]:
+            await self.send_table("Total scores so far:")
+        
+        table = ""
+        for word, score in self.valid_words.most_common():
+            claim = "none"
+            if word in self.claims:
+                claim = self.claims[word]
+            table += f"{word}\t{score}\t{claim}\n"
+            if len(table) > 1900:
+                await self.ctx.send(box(table, lang="diff"))
+                table = ""
+        if table != "":
+            await self.ctx.send(box(table, lang="diff"))
+    
+    async def timer_task(self):
+        reveal = 3
+        while reveal:
+            await asyncio.sleep(30)
+            msg = f"{30*reveal} seconds remaining in round {self.level+1}. {len(self.valid_words)-len(self.claims)} words left to find."
+            await self.send_round_table(msg)
+            reveal -= 1
+        await asyncio.sleep(30)
+
+    def check_message(self, message: discord.Message):
+        early_exit = message.channel != self.ctx.channel or message.author == self.ctx.guild.me
```

Comment:
Change the second condition in this to `message.author.bot` (which is a boolean).